trackintel/analysis/modal_split.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+import pandas as pd
+
+from trackintel.geogr.distances import check_wgs_for_distance_calculation, \
+    calculate_haversine_length
+
+
+def calculate_modal_split(tpls_in, freq=None, metric='count', per_user=False, norm=False):
+    """Calculate the modal split of triplegs
+
+    Parameters
+    ----------
+    tpls_in : GeoDataFrame (as trackintel triplegs)
+        triplegs require the column `mode`.
+    freq : str
```

Comment:
would be good to provide a link to pandas.Grouper or https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#offset-aliases to show the possible accepted values